netlify.toml

```diff
@@ -1,12 +1,9 @@
 [build.environment]
   NODE_VERSION = "18"
+  NODE_OPTIONS = "--max_old_space_size=4096"
```

Comment:
Why would we need to add this? I mean why would this cause a JavaScript heap out of memory?

Reply:
https://app.netlify.com/sites/eslint-stylistic/deploys/6521011f2f69ed0008616702
https://github.com/unocss/unocss/commit/b07065aa1f994c8f41ea888582abf2aa405ae3df
This problem has also occurred in unocss document packaging. I am not sure if it is a problem with vitepress.